src/components/SoftwareVersion.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {useEffect, useState} from "react";
+import SoftwareVersionFetcher from "../minecraft-versioning/SoftwareVersionFetcher";
+
+export default function SoftwareVersion({ versionType, project = "paper" }) {
```

Comment:
I don't know how `typescript` we want to go, but ideally the props would have types. 

I don't know if there is a better way than just defining a local interface like
```typescript
interface SoftwareVersionProps
	versionType: "maj-min" | "maj" | "min",
	project?: "paper" | "whatever",
}

export default function SoftwareVersion({ versionType, project = "paper" }: SoftwareVersionProps) {
}
```